common.sh

```diff
@@ -2,18 +2,33 @@
 CACHE=/var/cache/preupgrade
 PREUPGRADE_CACHE=/var/cache/preupgrade/common
 PREUPGRADE_CONFIG=/etc/preupgrade-assistant.conf
+
 VALUE_RPM_QA=$PREUPGRADE_CACHE/rpm_qa.log
+PREUPG_RPM_QA=$VALUE_RPM_QA
 VALUE_ALL_CHANGED=$PREUPGRADE_CACHE/rpm_Va.log
+PREUPG_ALL_CHANGED=$VALUE_ALL_CHANGED
 VALUE_CONFIGCHANGED=$PREUPGRADE_CACHE/rpm_etc_Va.log
+PREUPG_CONFIGCHANGED=$VALUE_CONFIGCHANGED
 VALUE_PASSWD=$PREUPGRADE_CACHE/passwd.log
+PREUPG_PASSWD=$VALUE_PASSWD
 VALUE_CHKCONFIG=$PREUPGRADE_CACHE/chkconfig.log
+PREUPG_CHKCONFIG=$VALUE_CHKCONFIG
 VALUE_GROUP=$PREUPGRADE_CACHE/group.log
+PREUPG_GROUP=$VALUE_GROUP
 VALUE_RPMTRACKEDFILES=$PREUPGRADE_CACHE/rpmtrackedfiles.log
+PREUPG_RPMTRACKEDFILES=$VALUE_RPMTRACKEDFILES
 VALUE_RPM_RHSIGNED=$PREUPGRADE_CACHE/rpm_rhsigned.log
+PREUPG_RPM_RHSIGNED=$VALUE_RPM_RHSIGNED
 VALUE_ALLMYFILES=$PREUPGRADE_CACHE/allmyfiles.log
+PREUPG_ALLMYFILES=$VALUE_ALLMYFILES
 VALUE_EXECUTABLES=$PREUPGRADE_CACHE/executable.log
+PREUPG_EXECUTABLES=$VALUE_EXECUTABLES
 VALUE_TMP_PREUPGRADE=$XCCDF_VALUE_TMP_PREUPGRADE
+PREUPG_TMP_PREUPGRADE=$VALUE_TMP_PREUPGRADE
+CONFIG_FILES_DIR=$PREUPG_TMP_PREUPGRADE/config_files
+
 POSTUPGRADE_DIR=$VALUE_TMP_PREUPGRADE/postupgrade.d
+PREUPG_POSTUPGRADE_DIR=$POSTUPGRADE_DIR
 CURRENT_DIRECTORY=$XCCDF_VALUE_CURRENT_DIRECTORY
```

Comment:
How do you think to get rid of them?
They has to exist in common.sh otherwise contents will not see them.